build.gradle

```diff
@@ -32,8 +32,8 @@ sourceCompatibility = 1.7
 
 group = "com.google.api"
 version = "0.0.0-SNAPSHOT"
-// url to the snapshot repository of a private maven repo.
-def toolkitSnapshotRepo = "http://104.197.230.53:8081/nexus/content/repositories/snapshots/"
+// Nexus repository for publishing toolkit snapshots and downloading api-compiler snapshot.
+def nexusSnapshotRepo = "https://oss.sonatype.org/content/repositories/snapshots/"
```

Comment:
@ethanbao Is there any reason why we need to continue publishing `toolkit` to the private Nexus repo vs. the official one? AFAIK, `artman` and `packman` are the only consumers of `toolkit`.